aidial_sdk/application.py

```diff
@@ -68,6 +75,49 @@ def add_chat_completion(
     ) -> None:
         self.chat_completion_impls[deployment_name] = impl
 
+    async def _rate_response(
+        self, deployment_id: str, original_request: Request
+    ) -> Response:
+        set_log_deployment(deployment_id)
+        impl = self.chat_completion_impls.get(deployment_id, None)
+
+        if not impl:
+            return JSONResponse(
+                status_code=404,
+                content=json_error(
+                    message="The API deployment for this resource does not exist.",
+                    code="deployment_not_found",
+                ),
+            )
+
+        try:
+            body = await original_request.json()
+        except JSONDecodeError as e:
+            return JSONResponse(
+                status_code=400,
+                content=json_error(
+                    message=f"Your request contained invalid JSON: {str(e)}",
+                    type="invalid_request_error",
+                ),
+            )
+        log_debug(f"request: {body}")
+        try:
+            request = RateRequest(
+                **body,
```

Comment:
If you remove the trailing comma, then Black with format these three lines into a single line: `RateRequest(**body)`.

Reply:
fixed